src/app/extensions/quickorder/shared/quickorder-repeat-form-quantity/quickorder-repeat-form-quantity.component.ts

```diff
@@ -52,7 +52,7 @@ export class QuickorderRepeatFormQuantityComponent implements OnInit, OnChanges,
   }
 
   private setContext() {
-    this.context.set('sku', () => ' ');
+    this.context.set('sku', () => '_');
```

Comment:
As far as I understand the problem: The sku is initialized here with something truthy, otherwise the `ish-product-quantity` component would not display (due to its `visible$` not being true, when there is no product). 
How about adding an `alwaysVisible` input to the `ish-product-quantity` and not initializing the sku here anymore.
That would also not fire any search for an sku anymore...

Reply:
Turns out, this can be done with context display properties. Also in here: #844